sys-utils/nsenter.c

```diff
@@ -450,6 +498,14 @@ int main(int argc, char *argv[])
 	if (do_uid || do_gid)
 		open_target_fd(&uid_gid_fd, "", NULL);
 
+	/*
+	 * Get parent userns from any available ns.
+	 */
+	if (do_user_parent) {
+		if (set_parent_user_ns_fd() < 0)
+			errx(EXIT_FAILURE, _("failed to set parent user namespace"));
```

Comment:
This code can never be reached because all `return -1` in `set_parent_user_ns_fd()` call `err()` which aborts the program before returning.

Reply:
Done